dev/release/03-binary.sh

```diff
@@ -50,26 +50,467 @@ if [ ! -d "$artifact_dir" ]; then
   exit 1
 fi
 
-tag=apache-arrow-${version}
-tagrc=${tag}-rc${rc}
+if [ -z "${BINTRAY_PASSWORD}" ]; then
+  echo "BINTRAY_PASSWORD is empty"
+  exit 1
+fi
+
+: ${BINTRAY_REPOSITORY:=apache/arrow}
+
+docker_run() {
+  local uid=$(id -u)
+  local gid=$(id -g)
+  docker \
+    run \
+    --rm \
+    --tty \
+    --interactive \
+    --user ${uid}:${gid} \
+    --volume "$PWD":/host \
+    ${docker_image_name} \
+    bash -c "cd /host && $*"
+}
+
+docker_run_gpg_ready() {
+  local gpg_agent_socket_dir="$(gpgconf --list-dir socketdir)"
+  local uid=$(id -u)
+  local gid=$(id -g)
```

Comment:
On osx it's 20 (staff:*:20:root) for me, which already exists in the docker container so groupadd fails with `groupadd: GID '20' already exists`. If I set `gid=uid` it works.